src/formTimingTracking.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Form timing tracking
+ * 
+ * This script analyses the interactions with <form> fields.
+ */
+
+ export default (subscribe, {formNameAttribute, fieldNameAttribute}) => {
+    let field_timings = {};
+    
+    const ppas_track_form_field_entry = function (e){
```

Comment:
you don't need to use prefixes like ppas, name this function `trackFormFieldEntry`

Reply:
use arrow function